app/components/avo/fields/file_field/edit_component.html.erb

```diff
@@ -6,11 +6,21 @@
   <% end %>
 
   <% if @resource.authorization.authorize_action(:upload_attachments?, raise_exception: false) %>
-    <%= @form.file_field @field.id,
-      data: @field.get_html(:data, view: view, element: :input),
-      direct_upload: @field.direct_upload,
-      disabled: @field.readonly,
-      style: @field.get_html(:style, view: view, element: :input)
-    %>
+    <% if @field.accept.present? %>
+      <%= @form.file_field @field.id,
+        accept: @field.accept,
+        data: @field.get_html(:data, view: view, element: :input),
+        direct_upload: @field.direct_upload,
+        disabled: @field.readonly,
+        style: @field.get_html(:style, view: view, element: :input)
+      %>
+    <% else %>
+      <%= @form.file_field @field.id,
+        data: @field.get_html(:data, view: view, element: :input),
+        direct_upload: @field.direct_upload,
+        disabled: @field.readonly,
+        style: @field.get_html(:style, view: view, element: :input)
+      %>
+    <% end %>
```

Comment:
I think we can pass the attribute to the field, can we not? If it's `nil` it will be ignored. If it has a value, it will be attributed, right?

```suggestion
    <%= @form.file_field @field.id,
      accept: @field.accept,
      data: @field.get_html(:data, view: view, element: :input),
      direct_upload: @field.direct_upload,
      disabled: @field.readonly,
      style: @field.get_html(:style, view: view, element: :input)
    %>
```